tiktok.cs

```diff
@@ -5,394 +5,105 @@ namespace First_project
     class Program
     {
             
-        static void XTurn(out int xGuessB, out int yGuessB)
+        static void Turn(out int xGuessB, out int yGuessB, int playerToken)
         {
             string xVal;
             string yVal;
-            Console.WriteLine("Player X! Are you ready?");
-            Console.WriteLine("What is the x coordinate of your guess?");
-            xVal = Console.ReadLine();
-            xGuessB = Convert.ToInt32(xVal);
-            Console.WriteLine("What is the y coordinate of your guess?");
-            yVal = Console.ReadLine();
-            yGuessB = Convert.ToInt32(yVal);
-
-        }
+            string player = playerToken == 1 ? "X" : "Y";
 
-
-        static void YTurn(out int xGuessB, out int yGuessB)
-        {
-            string xVal;
-            string yVal;
-            Console.WriteLine("Player Y! Are you ready?");
+            Console.WriteLine($"Player {player}! Are you ready?");
             Console.WriteLine("What is the x coordinate of your guess?");
             xVal = Console.ReadLine();
             xGuessB = Convert.ToInt32(xVal);
+            xGuessB--;
             Console.WriteLine("What is the y coordinate of your guess?");
             yVal = Console.ReadLine();
             yGuessB = Convert.ToInt32(yVal);
+            yGuessB--;
         }
 
         static void DrawBoard(int[,] board)
         {

```

Comment:
I added newlines here to make it a bit more readable, totally not necessary though.